pom.xml

```diff
@@ -89,22 +89,56 @@ under the License.
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <sourceReleaseAssemblyDescriptor>source-release</sourceReleaseAssemblyDescriptor>
     <gpg.useagent>true</gpg.useagent>
-    <minimalMavenBuildVersion>3.2.5</minimalMavenBuildVersion>
+    <minimalMavenBuildVersion>3.6.3</minimalMavenBuildVersion>
     <minimalJavaBuildVersion>1.8</minimalJavaBuildVersion>
     <maven.compiler.source>${maven.compiler.target}</maven.compiler.source>
     <maven.compiler.target>1.7</maven.compiler.target>
-    <surefire.version>3.1.0</surefire.version><!-- for surefire, failsafe and surefire-report -->
+    <surefire.version>3.1.2</surefire.version><!-- for surefire, failsafe and surefire-report -->
```

Comment:
Update should be done as separate PR and issue

Reply:
Sure, no problem. I'll back it out.